src/rpc/net.cpp

```diff
@@ -767,15 +769,18 @@ static RPCHelpMan listbanned()
 
     banmap_t banMap;
     node.banman->GetBanned(banMap);
+    const int64_t current_time{GetTime()};
```

Comment:
https://github.com/bitcoin/bitcoin/blob/dd01dc6a1d23e1f316940cd1092ad716216331aa/src/util/time.h#L40-L44

```suggestion
    const int64_t current_time{GetSystemTimeInSeconds()};
```

Reply:
banman uses `GetTime()` (mockable), so mixing non-mockable and mockable should be avoided

Reply:
Ah, thanks!

Reply:
So it should be `GetTime<std::chrono::seconds>`?

Reply:
I guess so, but that refactor should also be done for nCreateTime, banman, ...